guides/concepts/index.md

```diff
@@ -1 +1,100 @@
-From an outside perspective it is clear that a lot of the underlying concepts in Cardstack reflect some of the useful architectural decisions of Ember. If you are familiar with Ember then you are likely going to understand the most/all of these concepts without too much effort.
+This section is a crash course in Cardstack concepts. It includes vocabulary that you'll find throughout the other topics, and it draws some parallels to other frameworks and JavaScript in general.
+
+
+## Essential Vocabulary
+
+### Cards
+
+Cards are reusable units of functionality - like a blog article, a text editor, a payment system, and more.
+A developer would use many cards, and cards inside of cards, to create the end user experience.
+Each card can have its own data schema, which flows from the front end to the data layer. When someone is looking at a card, it is reflected in the browser URL.
+The information contained in that URL informs communication with the back end servers.
+
+### The Cardstack mono-repo
+
+The [Cardstack mono-repo](https://github.com/cardstack/cardstack) is a collection of the various elements that could be used to create a Cardstack environment.
```

Comment:
I'm a little unclear on what a "Cardstack environment" means. Maybe we can define that?

Reply:
Or does it just mean an "app" with `cardstack/cardstack` in the package.json?

Reply:
Oh I just saw the `Cardstack Environment` section below. Perhaps we should move it up and define it upfront before using the term?

Reply:
I'll call it "cardstack project" here for now.